neurodsp/spectral/power.py

```diff
@@ -106,7 +106,7 @@ def compute_spectrum_wavelet(sig, fs, freqs, avg_type='mean', **kwargs):
         freqs = create_freqs(*freqs)
 
     mwt = compute_wavelet_transform(sig, fs, freqs, **kwargs)
-    spectrum = get_avg_func(avg_type)(mwt, axis=0)
+    spectrum = get_avg_func(avg_type)(abs(mwt)**2, axis=1)
```

Comment:
This could perhaps use some in-code commentary. I'm thinking something like:

```
# Convert the wavelet coefficient outputs to power
mwt_power = abs(mwt)**2
# Create the power spectrum by averaging across the time dimension
spectrum = get_avg_func(avg_type)(abs(mwt)**2, axis=1)
```

My comments are totally ad-hoc / placeholders, in so far as I understand what's being done. If you think the description could be useful, please replace with your own that I'm sure will be more descriptive / useful!

Reply:
I think this is a great suggestion.